cfme/scripting/tests/test_quickstart.py

```diff
@@ -34,6 +34,8 @@ def yamls_volume():
     return volume
 
 
+@pytest.mark.tcpstack('all')
```

Comment:
we should use a named constant here instead of a magic value
perhaps it makes sense to just use the list of supported ones

an option would be to default to "all" if none are given